operator/cmd/run/run.go

```diff
@@ -170,7 +170,7 @@ func Command() *cobra.Command {
 			var cloudExpander *pkgsecrets.CloudExpander
 			if cloudSecretsEnabled {
 				cloudConfig := pkgsecrets.ExpanderCloudConfiguration{}
-				if cloudSecretsAWSRegion != "" && cloudSecretsAWSRoleARN != "" {
+				if cloudSecretsAWSRegion != "" {
 					cloudConfig.AWSRegion = cloudSecretsAWSRegion
 					cloudConfig.AWSRoleARN = cloudSecretsAWSRoleARN
```

Comment:
Can you add a comment noting that `cloudSecretsAWSRoleARN` being `""` is acceptable (and why). Otherwise this will look like a bug in the future.